docs/tutorial/application-distribution.md

```diff
@@ -1,24 +1,38 @@
 # Application Distribution
 
-To distribute your app with Electron, you need to package and rebrand it. The easiest way to do this is to use one of the following third party packaging tools:
+## Overview
+
+To distribute your app with Electron, you need to package and rebrand it.
+To do this, you can either use specialized tooling or manual approaches.
+
+## With Tooling
```

Comment:
I think on other rewritten docs, we've only capitalized the first letter in each header? Might want to change these for consistency.